11_Largest_Product_In_Grid.py

```diff
@@ -0,0 +1,101 @@
+# grid2 = [
```

Comment:
geh?

Reply:
its on my laptop, but i just used the previous isPrime() method i made and had it for each in range to 2 million. to count how many prime numbers there were.  It took forever but it works.

Reply:
oh before i decided i wanted to read it from a file, i just manually changed it to a list of list, but then i was afraid you would judge me for doing that.